0004-remove-spandata.md

```diff
@@ -0,0 +1,43 @@
+# Remove SpanData
```

Comment:
Please include the use-cases for this as well. See as an inspiration https://github.com/open-telemetry/opentelemetry-specification/issues/71#issuecomment-506083966